posthog/api/test/test_cohort.py

```diff
@@ -36,6 +36,10 @@ def test_creating_update_and_calculating(self, patch_calculate_cohort, patch_cap
             self.user,
             "cohort created",
             {
+                "filters": {
+                    "type": "OR",
```

Comment:
hmm, how do you imagine querying / using this info?

Reply:
`Contains` would be the best we get